python/fusion_engine_client/utils/log.py

```diff
@@ -179,6 +187,8 @@ def find_log_file(input_path, candidate_files=None, return_output_dir=False, ret
     @param log_base_dir The base directory to be searched when performing a pattern match for a log directory.
     @param check_exact_match If `True`, check if `input_path` is the path to a data file. Otherwise, skip this check and
            only perform a pattern search.
+    @param check_pattern_match If `True` and `input_path` does not refer to a log file or directory, perform a pattern
```

Comment:
This interface seems like it would be simpler if it assumed there might be a fuzzy match, and selected the exact match if there was an ambiguity. It seems like there isn't a strong use case for having the `check_exact_match`, and `check_pattern_match` arguments.